src/Card/CardImageCap.jsx

```diff
@@ -1,18 +1,45 @@
 import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import Skeleton from 'react-loading-skeleton';
 import CardContext from './CardContext';
 
 const CardImageCap = React.forwardRef(({
   src,
   srcAlt,
   logoSrc,
   logoAlt,
+  skeletonHeight,
+  skeletonWidth,
+  capLogoSkeleton,
+  capLogoSkeletonHeight,
+  capLogoSkeletonWidth,
   className,
 }, ref) => {
-  const { orientation } = useContext(CardContext);
+  const { orientation, isLoading } = useContext(CardContext);
   const wrapperClassName = `pgn__card-wrapper-image-cap ${orientation}`;
 
+  if (isLoading) {
+    return (
+      <div className={classNames(className, wrapperClassName)}>
+        <Skeleton
+          containerClassName="pgn__card-image-cap-loader"
+          height={skeletonHeight}
+          width={skeletonWidth}
+        />
+        {capLogoSkeleton
+          && !!logoSrc
```

Comment:
If data for the card is loading, there's a pretty good change `logoSrc` hasn't been returned by an API yet, and as such would be a null/undefined value at this point. I don't think we can rely on having a truthy value for `logoSrc` here because of this.

Reply:
Yes, it's true
I deleted logoSrc in this place
Thanks